iree/compiler/Dialect/LinalgExt/Transforms/ConvertToLoops.cpp

```diff
@@ -90,7 +90,11 @@ struct BubbleSortConversion : public OpRewritePattern<linalg_ext::SortOp> {
           b.create<scf::IfOp>(
               loc, TypeRange{}, cond,
               [&](OpBuilder& b, Location loc) {
-                // Swap the pairs if true.
+                // Do not swap the pairs if true.
```

Comment:
This leads me to believe the sort is not as generally defined. It should have worked with both "GT" and "LT". Is that not the case? This should have been resolved while lowering from mhlo.sort -> linalg_ext.sort.

Reply:
I misunderstood the definition. The old sort.mlir was wrong in the beginning and I did not notice it. (since it used to work for VMLA) We sorted the array in a reverse order. I can add test cases for both "GT" and "LT".

The operation semantics states: after the array is sorted, it holds for all index positions (i, j) where `comparator(v[i], v[j]) = true`. Thus, if it is true, we don't swap (i, j) pair.

Reply:
Ok, make sense. Adding a test for both "GT" and "LT" would be good.